src/cmd/linuxkit/initrd/initrd.go

```diff
@@ -3,13 +3,13 @@ package initrd
 import (
 	"archive/tar"
 	"bytes"
-	"compress/gzip"
 	"errors"
 	"io"
 	"io/ioutil"
 	"path/filepath"
 	"strings"
 
+	gzip "github.com/klauspost/pgzip"
```

Comment:
I think a comment here that it is a drop-in 100% compatible replacement and 17% faster than `compress/gzip` would be helpful.